internal/armada/repository/event_job_status.go

```diff
@@ -35,12 +74,13 @@ func (p *EventJobStatusProcessor) handleMessage(eventMessage *api.EventMessage)
 
 	switch event := event.(type) {
 	case *api.JobRunningEvent:
-		err = p.jobRepository.UpdateStartTime(event.JobId, event.ClusterId, event.Created)
+		eventMessage, err := api.Wrap(event)
 		if err != nil {
-			log.Errorf("error while updating job start time: %v", err)
-			if err.Error() != JobNotFound {
-				return err
-			}
+			log.Errorf("error when wrapping job running event: %v", err)
```

Comment:
What was the purpose of the deleted code which ignored JobNotFound?  Is it a hack that's no longer required?